ArchUnitNET/Fluent/Syntax/Elements/Members/MembersShouldThat.cs

```diff
@@ -0,0 +1,111 @@
+﻿using ArchUnitNET.Domain;
+using static ArchUnitNET.Fluent.Syntax.ActivatorHandler;
+
+namespace ArchUnitNET.Fluent.Syntax.Elements.Members
+{
+    public class MembersShouldThat<TRuleTypeShouldConjunction, TReferenceType, TRuleType> :
+        ObjectsShouldThat<TRuleTypeShouldConjunction, TReferenceType, TRuleType>,
+        IMembersThat<TRuleTypeShouldConjunction>
+        where TReferenceType : IMember
+        where TRuleType : ICanBeAnalyzed
+    {
+        protected MembersShouldThat(IArchRuleCreator<TRuleType> ruleCreator,
```

Comment:
the name is bit weird as is it not grammatically correct

Reply:
renamed to ShouldRelatedToMembersThat in new version